### DISCUSSION TITLE
Good antenna for Rnode device?

### DISCUSSION THREAD

**Discussion by kimusan** [markqvist/Reticulum#446] [Help & Questions]
Hi
I have the lillygo board with mounted antenna and want to use it in a 3d printed case. For this I need to change to use a pigtail antenna.
What antennas can you recommend for this? It needs to be able to fit in the 3d printed case from unsigned.io

**Comment by markqvist** [comment]
Any kind of u.FL-to-SMA pigtail [like this one](https://www.adafruit.com/product/851) should fit in the case. You can then use any SMA antenna tuned for your desired operating frequencies.
